src/index.js

```diff
@@ -1,26 +1,44 @@
 import './style.css';
-import Selector from '../modules/selectors.js';
-import Todo from '../modules/todo.js';
 
-const objectSelector = new Selector();
-const objectTodo = new Todo();
+const todoBody = document.querySelector('.todo_main_content');
 
-window.addEventListener('load', () => {
-  objectTodo.dispaly();
-});
-objectSelector.todoForm.addEventListener('submit', (e) => {
-  objectTodo.addTask(e);
-});
+const todos = [
+  {
+    index: 1,
+    description: 'Do Laundry',
+    completed: true,
+  },
+  {
+    index: 2,
+    description: 'Go shopping',
+    completed: false,
+  },
+  {
+    index: 3,
+    description: 'Clean house',
+    completed: true,
+  },
+  {
+    index: 4,
+    description: 'Walk the dog',
+    completed: false,
+  },
+];
```

Comment:
> Remove all hardcoded items from the tasks array.

- You have done great so far, but there are some requirements you still need to meet before moving to the next milestone. :+1: 
- [ ] According to the project requirement, you were meant to remove all hardcoded todo items but it seems you didn't. I'll suggest you adhere to the requirement and remove them. :smile: :wave:  

- [ ] I'll suggest you update the start script in your `package.json` file from `"start": "webpack ",` to this `"start": "webpack serve --open",` so that your server actually starts when the command is run instead of just building the project. :+1: :+1: